clients/pyethereum:develop/Dockerfile

```diff
@@ -9,6 +9,11 @@ RUN \
   python get-pip.py &&                                                         \
   git clone --depth 1 https://github.com/ethereum/pyethapp  && \
   cd pyethapp && python setup.py develop                                                   && \
```

Comment:
Right, since we have two commands that need to run inside pyethapp, I removed the parenthesis and got rid of the second "cd pyethapp"